packages/node/src/soroban/utils.soroban.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright 2020-2022 OnFinality Limited authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { ApiWrapper } from '@subql/types-soroban';
+
+export function calcInterval(api: ApiWrapper): number {
+  // TODO find a way to get this from the blockchain
```

Comment:
Was there any luck with this?

Reply:
It doesn't seem to be uniform: https://dashboard.stellar.org/
there were discussions in the past with the ledger time being 5s, but now it seems to have grown to 6s